apps/vth-frontend/src/app/[locale]/print/page.tsx

```diff
@@ -81,32 +83,41 @@ const Page = async ({ params: { locale } }: { params: { locale: string } }) => {
   });
 
   return (
-    <div className={'utrecht-print-page'}>
-      <PrintButton />
-      <Grid className={'utrecht-grid--content-padding'}>
-        <div className={'utrecht-grid__full-width'}>
-          <section>
-            <Heading1>{printPageData?.title}</Heading1>
-            <Emphasis>Versiedatum: {printPageData?.versiondate}</Emphasis>
-            <Paragraph>{printPageData?.introductionBody}</Paragraph>
-          </section>
-          <section>
-            <Heading2>Inhoudsopgave</Heading2>
-            <TableOfContents navigationPagesResponse={navigationPagesResponse} />
-          </section>
+    <Page>
+      <PageContent className="utrecht-custom-page-content">
+        <div className="utrecht-print-page">
+          <PrintButton />
+          <Grid spacing="lg">
+            <GridCell sm={12}>
+              <section>
+                <Heading1>{printPageData?.title}</Heading1>
+                <Emphasis>Versiedatum: {printPageData?.versiondate}</Emphasis>
+                <Paragraph>{printPageData?.introductionBody}</Paragraph>
+              </section>
+              <section>
+                <Heading2>Inhoudsopgave</Heading2>
+                <TableOfContents navigationPagesResponse={navigationPagesResponse} />
+              </section>
+            </GridCell>
+          </Grid>
+          <Grid spacing="lg">
+            {navigationPagesResponse.data[0] &&
+              navigationPagesResponse.data.map(({ attributes: navigationPage }, index) => {
+                return (
+                  <GridCell sm={12} key={index}>
+                    {NavigationPageDisplay(navigationPage, (index + 1).toString())}
+                  </GridCell>
+                );
+              })}
+          </Grid>
         </div>
-      </Grid>
-      {navigationPagesResponse.data[0] &&
-        navigationPagesResponse.data.map(({ attributes: navigationPage }, index) => {
-          return (
-            <Grid className={'utrecht-grid--content-padding'} key={index}>
-              <div className={'utrecht-grid__full-width'}>
-                {NavigationPageDisplay(navigationPage, (index + 1).toString())}
-              </div>
-            </Grid>
-          );
-        })}
-    </div>
+        <Grid spacing="lg">
+          <GridCell md={12} justifyContent="flex-end">
+            <ScrollToTopButton Icon={UtrechtIconChevronUp}>{t('actions.scroll-to-top')}</ScrollToTopButton>
+          </GridCell>
+        </Grid>
```

Comment:
@AliKdhim87 Hebben we extra print css nodig om ervoor te zorgen dat deze button niet zichtbaar is op de print versie?